TesseractOCR/build_dependencies.sh

```diff
@@ -7,8 +7,9 @@
 
 GLOBAL_OUTDIR="`pwd`/build"
 LOCAL_OUTDIR="./outdir"
-LEPTON_LIB="`pwd`/leptonica-1.71"
 TESSERACT_LIB="`pwd`/tesseract-3.03"
+LEPTON_LIB="`pwd`/leptonica-1.72"
+IMAGE_LIBS="`pwd`/libtiff-ios-master/dependencies"
```

Comment:
I will update readme file as soon as we'll merge this.

Also, in my oppinion the idea with pods is greate. But I'll better do it in another ticket. I'm not well familiar with pods, so... What's about you? Can you add that pod?

Also it seems a good idea to add tesseract and leptonica as pod links as well.